puppet/modules/leap_mx/manifests/init.pp

```diff
@@ -45,11 +47,12 @@
   #
 
   package {
-    'leap-mx':
-      ensure  => latest,
-      require => Class['site_apt::preferences::twisted'];
-
-    [ 'leap-keymanager' ]:
+    $sources['leap-mx']['package']:
+      ensure  => $sources['leap-mx']['revision'],
+      require => [
+        Class['site_apt::preferences::twisted'],
+        Class['site_apt::leap_repo'] ];
+    'leap-keymanager':
```

Comment:
we've been keeping different resources separated by a newline to make it more visually clear. its a minor nit, but this block in particular is dense, so having a newline between the leap-keymanager resource, and the leap-mx one would help unpack it.